src/app/fyle/add-edit-expense/camera-options-popup/camera-options-popup.component.ts

```
@@ -5,7 +5,7 @@
 import { PopupAlertComponent } from 'src/app/shared/components/popup-alert/popup-alert.component';
 import { MAX_FILE_SIZE } from 'src/app/core/constants';
 import { LoaderService } from 'src/app/core/services/loader.service';
-import { finalize, from, map, raceWith, switchMap, tap, timer } from 'rxjs';
+import { finalize, from, map, raceWith, shareReplay, switchMap, timer } from 'rxjs';
```

ESLint

[error] 'shareReplay' is defined but never used.

(@typescript-eslint/no-unused-vars)